test/e2e/libpod_suite_test.go

```diff
@@ -50,7 +37,7 @@ func (p *PodmanTestIntegration) setDefaultRegistriesConfigEnv() {
 func (p *PodmanTestIntegration) setRegistriesConfigEnv(b []byte) {
 	outfile := filepath.Join(p.TempDir, "registries.conf")
 	os.Setenv("REGISTRIES_CONFIG_PATH", outfile)
-	ioutil.WriteFile(outfile, b, 0644)
+	ioutil.WriteFile(outfile, b, 0654)
```

Comment:
What is this change about?

Reply:
i have no idea!  i didnt intentionally change it.